arkouda/dtypes.py

```diff
@@ -40,15 +41,38 @@
     "uint8": "{:n}",
     "np.float64": "f",
     "uint64": "{:n}",
+    "bigint": "{:n}",
 }
 
-dtype = np.dtype
+
+def dtype(x):
+    if (isinstance(x, str) and x == "bigint") or isinstance(x, BigInt):
+        return bigint
+    else:
+        return np.dtype(x)
```

Comment:
Just verifying my memory here. We are doing this because `BigInt` has a `dtype` of `object` in NumPy, correct? If that is the case, can we add a comment indicating that for clarity?

Reply:
that is right, every other arkouda dtype has an equivalent `np.dtype`. `Bigint` doesn't because numpy gives them dtype object. So we had to roll our own `bigint` type. I'll add a comment